src/Strategy/NuGet/ProjectAssetsJson.hs

```diff
@@ -68,52 +78,165 @@ analyze' file = do
       , dependencyManifestFiles = [file]
       }
 
-newtype ProjectAssetsJson = ProjectAssetsJson
-  { targets :: Map.Map Text (Map.Map Text DependencyInfo)
+data NuGetDep = NuGetDep
+  { depName :: Text
+  , depVersion :: Text
+  , completeDepType :: Text
+  , completeDeepDeps :: Map.Map Text Text
   }
-  deriving (Show)
+  deriving (Show, Eq, Ord)
 
-instance FromJSON ProjectAssetsJson where
-  parseJSON = withObject "ProjectAssetsJson" $ \obj ->
-    ProjectAssetsJson <$> obj .: "targets"
+data ProjectAssetsJson = ProjectAssetsJson
+  { targets :: Map.Map Text (Map.Map Text DependencyInfo)
+  , projectFramework :: Map.Map Text (Set Text)
+  }
+  deriving (Show, Eq, Ord)
```

Comment:
[nit] there are a lot of `Text` values floating around; it's probably worthwhile to `newtype` wrapper some (or all?) of these. it was hard to follow some of the type signatures below and make sure the right text values were used

for json decoding purposes, you can derive `FromJSON` on the newtypes:

```hs
newtype FrameworkName = FrameworkName { unFrameworkName :: Text }
  deriving (Eq, Ord, Show, FromJSON)
```